bin/terminus.php

```diff
@@ -34,15 +39,26 @@
 $output = new ConsoleOutput();
 $container = Robo::createDefaultContainer($input, $output, $application, $config);
 
+$container->share('request', Request::class);
+$container->inflector(RequestAwareInterface::class)
+    ->invokeMethod('setRequest', ['request']);
+
 $container->share('fileCache', FileCache::class);
+
 $container->share('session', Session::class)
     ->withArgument('fileCache');
 $container->inflector(SessionAwareInterface::class)
     ->invokeMethod('setSession', ['session']);
+
 $container->share('sites', Sites::class);
 $container->inflector(SiteAwareInterface::class)
     ->invokeMethod('setSites', ['sites']);
 
+// Add our models
+$container->add(User::class);
+// TODO: Add 21 more models :)
```

Comment:
Yeah. The runner didn't feel quite right. Maybe a new class is called for. Easy to change later though.